README.md

```diff
@@ -140,17 +145,24 @@ Examples:
 - `abc templates golden-test record --test-name=one_env,multiple_envs examples/templates/render/for_each_dynamic`:
   same as above, but only for the specific named tests.
 
-The `<test_name>` parameter gives the test names to record or verify, if not
+For `record` and `verify` subcommand, the `<test_name>` parameter gives the test names to record or verify, if not
 specified, all tests will be run against. This flag may be repeated, like
 `--test-name=test1`, `--test-name=test2`, or `--test-name=test1,test2`.
 
-The `<location>` parameter gives the location of the template.
+For `new-test` subcommand, the `<test_name>` parameter gives the test name used to create the goldentest, like
+`--test-name=test1`, `--test-name=test2`.
```

Comment:
For `new-test`, the test name isn't a flag, it's a positional argument. So I don't think this paragraph should say `--test-name`.

Reply:
Done